gfw_pixetl/tiles/raster_src_tile.py

```diff
@@ -370,17 +372,14 @@ def _windows(self, dst: DatasetWriter) -> Iterator[Window]:
                 window = self._union_blocks(dst, i, j, max_i, max_j)
                 try:
                     yield snapped_window(window.intersection(self.intersecting_window))
-                except rasterio.errors.WindowError as e:
-                    if "Bounds and transform are inconsistent" in str(e):
-                        # FIXME: This check was introduced recently in rasterio
-                        # Figure out what it means to fail, and fix the window
-                        # generating code in this function
-                        LOGGER.warning(
-                            f"Bogus window generated for tile {self.tile_id}! "
-                            f"i: {i} j: {j} max_i: {max_i} max_j: {max_j} window: {window}"
-                        )
-                    elif not (str(e) == "windows do not intersect"):
-                        raise
+                except rasterio.errors.WindowError:
```

Comment:
Does upgrading make this unnecessary?

Reply:
Hmm, good question. This change was actually made over a year ago because we seemed to be getting what were allegedly invalid windows (according to newer versions of rasterio). It seems that I thought this was inevitable and at least sometimes Mostly Harmless. rasterio has changed a lot since then. I'll back this particular change out, making the exception fatal, and see if it comes up.